test/gaussiannumbers/fmpzi-test.jl

```diff
@@ -211,7 +211,7 @@ end
    @test b == A && a == B
 end
 
-function test_elem(R::FlintZZiRing)
```

Comment:
Why does this line get changed?

Otherwise everything looks good.

Reply:
Copy & past error on my part.